src/Angular.js

```diff
@@ -893,7 +893,35 @@ var csp = function() {
   return (csp.isActive_ = active);
 };
 
+/**
+ * @ngdoc directive
+ * @name ng.directive:ngJq
+ *
+ * @element html
+ * @param {string} (optional) the name of the library available under `window` to be used for angular.element
```

Comment:
@mboudreau It's not there; I'm looking at the "Files changed" tab, not the inline comments view. I've also re-downloaded the PR and the change is not there.

Maybe you forgot to push the changes but, believe me, they're not here.

Also, would you be able to rebase the PR over latest master so that we can see the tests run?